### DISCUSSION TITLE
how to create a relationship between two subclasses B and C, both classes extend A

### DISCUSSION THREAD

**Discussion by ygbadamosi662** [sqlalchemy/sqlalchemy#10062] [Usage Questions]
im trying to create a one(B) to many(C) relationship and keep getting foreign key errors, i need help

**Comment by zzzeek** [comment]
what do your tables look like?

**Reply by zzzeek** [reply]
here's an example here in the test suite https://github.com/sqlalchemy/sqlalchemy/blob/8fc9e38c375bb851792e3521b141edfdf8aa83e1/test/orm/inheritance/test_relationship.py#L2846-L2865   , that's using the more legacy style but the main idea is the relationship includes [primaryjoin](https://docs.sqlalchemy.org/en/20/orm/join_conditions.html#configuring-how-relationship-joins)
